cli/cmd/install.go

```diff
@@ -419,13 +419,29 @@ func render(w io.Writer, values *l5dcharts.Values, stage string, options valuesp
 // resource is not part of the Helm chart and will not be present when installing
 // with Helm. If stringData is set to true, the secret will be rendered using
 // the StringData field instead of the Data field, making the output more
-// human readable.
-func renderOverrides(values *l5dcharts.Values, namespace string, stringData bool) ([]byte, error) {
+// human readable. If namespace is empty, The same is read from the passed
+// Values
+func renderOverrides(values chartutil.Values, namespace string, stringData bool) ([]byte, error) {
```

Comment:
Any reason not to just remove the namespace parameter here and always read the namespace from the values?

Reply:
That makes sense. Removed it.